ci/pipelines/pr.yml

```diff
@@ -492,7 +506,8 @@ resources:
     icon: github
     source:
       uri: https://github.com/alphagov/pay-ci
-      branch: master
+      # branch: master
+      branch: pp-9338-run-pr-e2e-tests-in-codebuild
```

Comment:
Should this be `master`?

Reply:
Indeed it should 🤦 I didn't mean to include this in my commit! Doh!

Reply:
Fixed